ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -54,7 +55,26 @@
     "ipahealthcheck.ipa.roles",
     "ipahealthcheck.ipa.topology",
     "ipahealthcheck.ipa.trust",
+    "ipahealthcheck.ipa.meta",
+    "ipahealthcheck.meta.core",
     "ipahealthcheck.meta.services",
+    "ipahealthcheck.system.filesystemspace",
+]
+
+sources_0_4 = [
+    "ipahealthcheck.dogtag.ca",
+    "ipahealthcheck.ds.replication",
+    "ipahealthcheck.dogtag.ca",
+    "ipahealthcheck.ipa.certs",
+    "ipahealthcheck.ipa.dna",
+    "ipahealthcheck.ipa.idns",
+    "ipahealthcheck.ipa.files",
+    "ipahealthcheck.ipa.host",
+    "ipahealthcheck.ipa.roles",
+    "ipahealthcheck.ipa.topology",
+    "ipahealthcheck.ipa.trust",
```

Comment:
"ipahealthcheck.meta.core" is also present with ipa-healthcheck 0.4

Reply:
added to ipa-healthcheck 0.4 as well.